app/components/inputs/PathInput.js

```diff
@@ -0,0 +1,18 @@
+class PathInput extends React.Component {
```

Comment:
Is it possible for PathInput be just a functional component over the existing `Input` (or maybe `TextInput`)? What exactly exists on PathInput that doesn't in Input/TextInput?

Reply:
I tried to use TextInput, but I was not able to override the flex properties by passing in a className - it insisted on taking up the whole container. I didn't feel comfortable modifying the Input class and breaking other stuff ;)

Reply:
ok.

Reply:
Maybe take a look at the Input class and try overriding the flex  properties if you get a second and I'll modify my code if you can get it working...

Reply:
I believe we can leave that for a separate PR (unifying/improving the css of the existing input and the pathInput) as it will need a review over all input instances, so you can leave it the way you implemented for the moment.